dotnet/src/Connectors/Connectors.HuggingFace/TextGeneration/TextGenerationStreamResponse.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text.Json.Serialization;
+
+#pragma warning disable CA1812
+
+namespace Microsoft.SemanticKernel.Connectors.HuggingFace.Core;
+
+internal sealed class TextGenerationStreamResponse
+{
+    [JsonPropertyName("index")]
+    public int Index { get; set; }
+
+    [JsonPropertyName("token")]
+    public TextGenerationToken? Token { get; set; }
+
+    [JsonPropertyName("generated_text")]
+    public string? GeneratedText { get; set; }
+
+    [JsonPropertyName("details")]
+    public string? Details { get; set; }
+
+    internal sealed class TextGenerationToken
+    {
+        [JsonPropertyName("id")]
+        public int Id { get; set; }
+
+        [JsonPropertyName("text")]
+        public string? Text { get; set; }
+
+        [JsonPropertyName("logprob")]
```

Comment:
Odd this isn't `log_prob`, cannot find the reference for this could you add it as a link?

Would be useful to do this for various request and response models.

Reply:
Unfortunately there's not documentation describing the Streaming Chunk object in Hugging Face documentation. 

This is the result I have using postman.

![image](https://github.com/microsoft/semantic-kernel/assets/19890735/35823a63-11ac-4a2a-9547-641cd28db56f)